Content.Server/_DV/Station/Components/StationSecondaryCentcommComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Server._DV.Station.Systems;
+using Robust.Shared.Map;
+using Robust.Shared.Utility;
+
+namespace Content.Server._DV.Station.Components;
+
+/// <summary>
+/// Spawns the non-emergency shuttle Central Command for a station on the same map as the <see cref="StationCentcommComponent"/>
+/// </summary>
+[RegisterComponent, Access(typeof(StationSecondaryCentcommSystem))]
+public sealed partial class StationSecondaryCentcommComponent : Component
+{
+    /// <summary>
+    /// The grid to load as secondary Central Command
+    /// </summary>
+    [DataField]
+    public ResPath Path = new ResPath("/Maps/_DV/centcomm.yml");
```

Comment:
'new' expression can be simplified
```suggestion
    public ResPath Path = new("/Maps/_DV/centcomm.yml");
```